client/src/modules/account_reference/search.modal.js

```diff
@@ -78,27 +79,7 @@ function ReferenceSearchModalController(ModalInstance, filters, Store, util, Pat
       displayValues.number = null;
     }
 
-    // push all searchQuery values into the changes array to be applied
-    angular.forEach(vm.searchQueries, (value, key) => {
-      if (angular.isDefined(value)) {
-
-        // To avoid overwriting a real display value, we first determine if the value changed in the current view.
```

Comment:
I am a little worried there is a corner case we are missing here, but I cannot reproduce it.  I remember that we wrote this code for a reason.  

Still, I don't think it is blocking.